envs/monkey_zoo/packer/setup_snmp_agent.yml

```diff
@@ -0,0 +1,38 @@
+---
+- name: Configure SNMP agent
+  hosts: all
+  remote_user: root
+  vars:
+    community_string: c0mmun1ty
+
+  tasks:
+    - name: Install SNMP
+      ansible.builtin.apt:
+        name:
+          - snmp=5.7.3+dfsg-1.8ubuntu3
+          - snmpd=5.7.3+dfsg-1.8ubuntu3
+          - libsnmp30=5.7.3+dfsg-1.8ubuntu3
```

Comment:
It might be wise to leave a comment about why these specific versions were chosen.

Reply:
Good idea